PilotDataPlatform/metadata
# PILOT-1556: Add date-range filtering to items/search endpoint

## Summary

- Add date-range filtering for last updated time of an item to `/items/search` endpoint (**last_updated_start**; **last_updated_end**)
- `container_code` parameter is now optional, in order to filter by date-range across all projects.

## JIRA Issues

- https://indocconsortium.atlassian.net/browse/PILOT-1556

## Type of Change

Please delete options that are not relevant.


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Query for items using `/items/search` endpoint  by `last_updated_start` and `last_updated_end`.
2. Ensure number of items returned in response matches the number of items last updated between those date ranges specified in the previous step.
